ironic/drivers/modules/redfish/utils.py

```diff
@@ -274,3 +278,100 @@ def _get_system():
                       'node %(node)s. Error: %(error)s',
                       {'address': driver_info['address'],
                        'node': node.uuid, 'error': e})
+
+def _ping_host(host):
+    response = subprocess.call(
+        "ping -c 1 {} 2>&1 1>/dev/null".format(host), shell=True)
+    return (response == 0)
+
+def wait_for_host_state(host,
+                        alive=True,
+                        ping_count=3,
+                        retries=24):
+    if alive:
+        ping_type = "pingable"
+
+    else:
+        ping_type = "not pingable"
+
+    LOG.info("Waiting for the iDRAC to become %s", ping_type)
+
+    response_count = 0
+    state_reached = False
+
+    while retries > 0 and not state_reached:
+        response = _ping_host(host)
+        retries -= 1
+        if response == alive:
+            response_count += 1
+            LOG.debug("The iDRAC is %s, count=%s",
+                      ping_type,
+                      response_count)
+            if response_count == ping_count:
+                LOG.debug("Reached specified ping count")
```

Comment:
Need to include the iDRAC IP in this logging statement

Reply:
Done, Logged iDRAC IP address while logging.